components/atom/switch/src/index.js

```diff
@@ -0,0 +1,193 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import AtomLabel from '@s-ui/react-atom-label'
+
+const BASE_CLASS = 'sui-AtomSwitch'
+
+const SIZES = {
+  MEDIUM: 'medium',
+  LARGE: 'large'
+}
+
+const TYPES = {
+  TOGGLE: 'toggle',
+  SELECT: 'select',
+  SINGLE: 'single'
+}
+
+class AtomSwitch extends Component {
+  constructor (props) {
+    super(props)
+
+    const {size, disabled, labelLeft, labelRight, type, name, label, labelOptionalText, onToggle} = props
+    this.onToggle = onToggle
```

Comment:
Avoid this. It's fine to just access `this.props` in your component and even deconstruct it, instead of copying every prop in the class as a property.